docs/api.rst

```diff
@@ -2,4 +2,9 @@ API documentation
 ~~~~~~~~~~~~~~~~~
 
 .. automodule:: pytket.extensions.pysimplex
-    :members: SimplexBackend
+.. automodule:: pytket.extensions.pysimplex._metadata
```

Comment:
Should this be here?

Reply:
Coverage checks report a failure if it is not included somewhere. As there is no file docstring, it will not render anything on the webpage so it is relatively inert